avm/res/signal-r-service/signal-r/main.bicep

```diff
@@ -452,10 +449,10 @@ type privateEndpointType = {
   customNetworkInterfaceName: string?
 
   @description('Optional. Specify the type of lock.')
-  lock: lockType
+  lock: lockType?
```

Comment:
Any reason for not importing the private endpoint type?

Reply:
Not at all. I simply missed the whole Private Endpoint type. Thanks for catching this 🙏🏻